cmd/vicadmin/vicadm_test.go

```diff
@@ -62,6 +62,75 @@ func init() {
 
 	config.hostCertFile = "fixtures/vicadmin_test_cert.pem"
 	config.hostKeyFile = "fixtures/vicadmin_test_pkey.pem"
+
+}
+
+func testLoginFailure(t *testing.T) {
```

Comment:
Don't tests need to be public in order for them to run with `go test`?  In any case, it's just a test.  Might as well make it public.